#!/usr/bin/env python

# remember to run the collofing command to make this script executable
# chmod a+x cfg/script_name.py

# Name of containing package
PACKAGE = 'birds_eye'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

# Numeric values using sliders
# General syntax is:
#   gen.add('var_name', type, 0, 'var_name', default, low, high)
gen.add('dist_cap',  double_t,   0, 'dist_cap',   2.0, 0.0, 5.0)
gen.add('drive_speed',  double_t,   0, 'drive_speed',   0.7, 0.0, 3.0)
gen.add('turn_multip',  double_t,   0, 'turn_multip',   1.0, -100.0, 100.0)


# gen.generate(PACKAGE, 'package_name', 'PascalCaseNodeName')
exit(gen.generate(PACKAGE, 'birds_eye', 'AvoidObstacle'))
